### PR TITLE
[WIP] Add the A3 Powered Suit

### PR DESCRIPTION
**Pull request by ghost**
:cl:
add: Added the A3 Powered Suit. Nukies can get it for 100 TC. It has a buncha guns, and looks cool.
/:cl:

inEe4yx9sA

### CODE REVIEW

**Comment by Quartich** [comment]
Damn that's hot

**Comment by yoyobatty** [comment]
What's that? make nanosuit 2?

**Comment by GuyonBroadway** [comment]
Syndicate agents test A3 prototypes in  secret Dubai test grounds colorized 2015.


https://www.bbc.co.uk/newsround/32718293

**Comment by diegoxt45** [comment]
cool and good +1

**Comment by diegoxt45** [comment]
why the fuck was my comment marked as spam?

**Comment by ghost** [comment]
@diegoxt45 because it was

File: hippiestation/code/modules/a3/_aaa.dm [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+				stat("Toxin Levels:", "[getToxLoss()]")
+				stat("Burn Severity:", "[getFireLoss()]")
+				stat("Brute Trauma:", "[getBruteLoss()]")
+				stat("Cellular Damage:", "[getCloneLoss()]")
+				stat("Stamina:", "[100 - getStaminaLoss()]%")
+				stat("Radiation Levels:","[radiation] rads")
+				stat("Body Temperature:", "[bodytemperature-T0C] degrees C ([bodytemperature*1.8-459.67] degrees F)")
+				stat("Atmospheric Pressure:","[pressure] kPa")
+				stat("Atmoshperic Temperature:","<span class='[environment.temperature > FIRE_IMMUNITY_MAX_TEMP_PROTECT?"alert":"info"]'>[round(environment.temperature-T0C, 0.01)] &deg;C ([round(environment.temperature, 0.01)] K)</span>")
+				stat("Atmospheric Thermal Energy:","THERMAL_ENERGY(environment)/1000] kJ")
```
Comment (yoyobatty) [inline hippiestation/code/modules/a3/_aaa.dm:180]: hmmm

**Comment by ghost** [comment]
ill finish this some other time

**Pull request closed without merging**